### ISSUE TITLE
not working anymore

### ISSUE THREAD

**Issue by 3xploiton3** [sondreb/Downloadr#48]
need fix to webstore

**Comment by sondreb** [comment]
Yes, Google have "killed" their web store for regular apps, Flickr Downloadr will be migrated to become a regular desktop app in a future release.
